shared/desktop/package.desktop.tsx

```diff
@@ -53,8 +54,13 @@ const packagerOpts: Options = {
   darwinDarkModeSupport: true,
   dir: desktopPath('./build'),
   download: {
-    mirrorOptions: {
-      mirror: 'https://kbelectron.keybase.pub/electron-download/',
```

Comment:
instead of hosting this all the time we just ensure the hash matches what we expect it to